en/hubs_tos.md

```diff
@@ -0,0 +1,82 @@
+Your Payment
+
+__Payment.__ We offer the Service as an automatically renewing subscription service. When you sign up, you authorize us to charge the payment method you provide for the subscription fees for the first term of the Service. Your plan renews automatically at the end of each term, and you authorize us to charge the subscription fees each term. If you stop paying, we will immediately suspend your account, and we may delete it after 2 months of not paying.
+
+__14-Day Refund Policy.__ The first time you subscribe to the Service through Mozilla's website, if you cancel your account within the first 14 days, you may request a refund and Mozilla will refund your first subscription term. This offer only applies the first time you subscribe. This is in addition to any rights you may have under your country’s consumer laws.
+You May Cancel Your Subscription at Any Time
+Cancellation. You may cancel your subscription to the Service at any time by deactivating your account and canceling your subscription. You can do this by clicking the “Cancel Subscription” link in any email that we send you about the Service or by going to the Subscriptions & Payments section of your Firefox Account and clicking “Cancel.” If you choose to cancel, auto-renewal will stop, we will not charge you for any future payment periods, and you will lose access to the Services at the end of the current billing cycle.
+
+## 2. Communications and Content
+Hubs allows your users to send information (such as audio, video, text, images, 3D models, and scenes) to other users.
+
+By signing up for and creating your Hubs, you agree to give Mozilla all rights necessary to operate Hubs. You also assure Mozilla that you have, or have obtained, the rights and authority necessary to grant Mozilla all rights and permissions necessary for the operation of Hubs. This includes, but is not limited to, a license and permission to process, transmit, and display the information you and your users send through Hubs. It also includes permission to gather and share information as described in the [Hubs Privacy Notice](https://www.mozilla.org/privacy/hubs-subscription/).
+
+When you and your users submit information to Hubs, the original creator of that information continues to own the rights to their content. You grant Mozilla a worldwide, royalty-free, perpetual, irrevocable, non-exclusive, transferable, and sublicensable license to use, copy, modify, adapt, prepare derivative works from, distribute, perform, and display that information, audio, video, images, or 3D models for the purpose of operating Hubs. You also agree that we may remove metadata associated with the information or data you submit. If you or one of your users allows remixing of a scene or avatar, you agree to license that scene or avatar under a [CC-BY 3.0](https://creativecommons.org/licenses/by/3.0/legalcode) license.
+To learn more about how Hubs operates, you can see the [source code](https://github.com/mozilla/hubs). 
```

Comment:
Add a line break between the paragraphs.

Reply:
fixed